### PR TITLE
Preparing for October Unami Update

### PR DESCRIPTION
**Pull request by dwgibbons**

### CODE REVIEW

File: _posts/news/2019-10-14-2019-Lodge-Leadership-Development.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+author: Eoghan Pleis
+date: 2019-10-14
+banner:
+thumbnail: sample.png
+excerpt: Develop your leadership skills while taking a leading role in developing the Lodge's future.
+---
+
+## December 6-8, 2019
+
+This Lodge Leadership Development (LLD) will not be one to miss! On the heals of [National OA's committment to Thrive](https://oa-bsa.org/article/thrive-national-webinar-local-commitment), we will be tackling the ambitious answer to "What's Next?" Many hands make light work, and getting many voices in on this discussion will surely help move the conversation forward.
```
Comment (crossfireprod) [inline _posts/news/2019-10-14-2019-Lodge-Leadership-Development.md]: * heels?

File: _posts/news/2019-10-14-2019-Lodge-Leadership-Development.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Many of the conversations and trainings will help participants learn how to better support the Lodge, Scouting, and many other communities they may find themselves a part of.
+
+More information will be coming shortly for registration, so stay tuned!
```
Comment (crossfireprod) [inline _posts/news/2019-10-14-2019-Lodge-Leadership-Development.md]: Do we at least have a general location?

File: _posts/news/2019-10-14-2019-New-Members-Bash.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+If you answered "yes" to any of the above questions then the New Members Bash is right for you!
+
+The new members' bash will have food and activities to keep you warm on a Novemember Friday evening. Representatives from every chapter and committee will be in attendance to help welcome you to the Lodge as well as answer any questions and help you get more involved.
+
+Can't wait till then? Reach out to [anyone from our team](/contactlist) to see how you can get more involved.
+
+More information will be coming in shortly as plans are finalized, but mark your calendars and save the date!
+
+## Location
+<iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d12198.050759720843!2d-75.2205455!3d40.1531374!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xca8343ab59710ae!2sBorough%20of%20Ambler!5e0!3m2!1sen!2sus!4v1571096129575!5m2!1sen!2sus" width="100%" height="400" frameborder="0" class="mt-3" allowfullscreen></iframe>
```
Comment (crossfireprod) [inline _posts/news/2019-10-14-2019-New-Members-Bash.md]: LGTM

File: chapters-find.md [hunk 4]
```diff
@@ -15,50 +15,50 @@ The best way to find what chapter you belong to is learning from your troop. You
...
+const chapters_ul = document.getElementById("findchapter-chapters");
+input.addEventListener('input', updateZip);
 
 var chapters_matched =[];
-var multiple_text = "The zipcode you entered aligns with the chapters below. Either use the map to see which chapter you are in or contact one of the leaders from your unit or chapter.";
+var multi_chapter_output = "";
 
-function updateValue(e) {
-  if(e.target.value.toString().length > 4)
+function updateZip(zip_input) {
```
Comment (crossfireprod) [inline chapters-find.md]: There's a human factors issue here, in that I get zero feedback if I enter an invalid zip.  That said, I'd add this to an issue list and re-visit ASAP - not keep it from going live.

File: _posts/news/2019-10-14-2019-Lodge-Leadership-Development.md (hunk 1 above)
Comment (dwgibbons) [reply _posts/news/2019-10-14-2019-Lodge-Leadership-Development.md]: Thanks fixed.

File: _posts/news/2019-10-14-2019-Lodge-Leadership-Development.md (hunk 2 above)
Comment (dwgibbons) [reply _posts/news/2019-10-14-2019-Lodge-Leadership-Development.md]: We do. I wasn't trying to make the location bit a standard article thing but I guess it makes sense when the article is explicitly about an event.

Comment (dwgibbons) [reply _posts/news/2019-10-14-2019-Lodge-Leadership-Development.md]: This as fixed now as well

File: _posts/news/2019-10-14-2019-Lodge-Leadership-Development.md [hunk 5]
```diff
@@ -15,3 +15,6 @@ This Lodge Leadership Development (LLD) will not be one to miss! On the heels of
 Many of the conversations and trainings will help participants learn how to better support the Lodge, Scouting, and many other communities they may find themselves a part of.
 
 More information will be coming shortly for registration, so stay tuned!
+
+## Location
```
Comment (crossfireprod) [inline _posts/news/2019-10-14-2019-Lodge-Leadership-Development.md]: :+1:

**Pull request merged**